### ISSUE TITLE
Reolink Doorbell WiFi: 2 way audio not working

### ISSUE THREAD

**Issue by ReX1983** [AlexxIT/WebRTC#713]
I can't find a way to make two-way audio work.

I have read all the past issues, questions and FAQs, but nothing worked. Would appreciate some guidance

My setup:
- HASS 2024.6.3 on Raspberry PI 4
- External access via Duck DNS and NGINX (public dynamic IP)
- WebRTC 3.5.2 installed as HASS custom integration (via HACS)
- Reolink Doorbell Wifi - firmware 3.0.0.3215. UPnP, RSTP and ONVIP enabled
- Port forwarding enabled for port 8555 on my router

This is my go2rtc yaml:
```
doorbell_2way:
    - rtsp://USER:PWD@IP:554/Preview_01_sub
    - "ffmpeg:doorbell_2way#audio=opus#audio=volume#async"

ffmpeg:
  bin: ffmpeg
  volume: "-af \"volume=25dB\""
  
webrtc:
  candidates:
    - stun:8555
```

This is my HA UI card:

```
type: custom:webrtc-camera
streams:
  - url: doorbell_2way
    mode: webrtc
    media: video,audio
    name: 🔇
  - url: doorbell_2way
    mode: webrtc
    media: video,audio,microphone
    name: 🔈
ui: true
muted: false
```

I have tried both desktop (chrome) and mobile (android chrome and HA companion app) and both from local access (LAN IP) and remote (HTTPS).

When I switch to the mic-on profile, I can see the browser either asking or displaying access to the microphone, but no sound is reproduced at the doorbell end.

I have tried multiple configs and navigated all the previous issues and threads on the topics. 

I have seen a reference to reverse proxy ([here](https://github.com/AlexxIT/WebRTC/issues/582#issuecomment-1767052984)) but even if I needed it, I am surprised it does not even work in local.

Thanks

**Comment by AlexxIT** [comment]
https://github.com/AlexxIT/go2rtc/issues/331

**Comment by ReX1983** [comment]
thanks @AlexxIT - the issue was the doorbell firmware, had to downgrade.

FYI - I have found a workaround to make two way audio working also on non HTTPS addresses (see [here](https://stackoverflow.com/questions/52759992/how-to-access-camera-and-microphone-in-chrome-without-https)). 

Tested and it works on both Chrome and Chrome mobile.

Unfortunately I can't seem to find a way to access the flags page from the Android companion app.
